tests/integration/browser.sqlite.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// mock tests for the Cordova SQLite Plugin
+
+var adapters = ['local'];
```

Comment:
If its only designed for one adapter, it doesnt need to do the adapters.forEach dance